lib/tests/formulae.rb

```diff
@@ -575,11 +578,11 @@ def formula!(formula_name, args:)
         fetch_args << "--force" if args.cleanup?
 
         new_formula = @added_formulae.include?(formula_name)
-        audit_args = [formula_name, "--online"]
+        audit_args = [formula_name, "--online", "--skip-style"]
```

Comment:
We shouldn't `--skip-style` for new formulae. This means FormulaAuditStrict cops will never be checked for new formulae.